### PR TITLE
Java-N experiment

### PR DESCRIPTION
**Pull request by nicovank**

### CODE REVIEW

File: scripts/output/Java.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+2.8529322147369385,24.034013519626797,15.316507816314697,22.893907732683967,1.3124361038208008,21.420091699672156,4.734622955322266,21.816752291404136,4.5859129428863525,16.021402763634335,,,6.792860269546509,22.35214976645725,0.683119535446167,19.93753134398627,1.560814619064331,17.429789487230874
+2.798633337020874,24.587820533201775,15.30681562423706,22.774618025164525,1.3239891529083252,19.695817602442354,4.756854295730591,20.240971361074095,4.600765943527222,14.296362843345607,,,6.8608434200286865,22.39377364355908,0.6578552722930908,19.934888877077622,1.5429320335388184,17.48354745860633
+2.776782989501953,24.700198526139637,15.264526605606079,22.793012849672614,1.3181114196777344,21.107080347024556,4.754719972610474,21.766012914383435,4.577277183532715,15.838509843151732,,,6.889050722122192,22.029910335177412,0.6290786266326904,20.31399960954798,1.6898670196533203,16.077246884820326
```
Comment (nicovank) [inline scripts/output/Java.csv:11]: Let's leave this data file out for now.

File: Java-N/reverse-complement/revcomp.java.in [hunk 2]
```diff
@@ -175,7 +178,8 @@ public void run()
          while (tob[p1] != '\n') p1++;
 
          int p2 = positions[3];
-      
+         // System.err.println(lp1 + " " + lp2 + " " + p1 + " " + p2);
+         // System.err.println(list.toString());
```
Comment (nicovank) [inline Java-N/reverse-complement/revcomp.java.in:182]:
```suggestion
```

**Review by nicovank — COMMENTED** [review]
This is great, thanks.

**Review by hyukjekwon — COMMENTED** [review]
Reviewed.

**Review by hyukjekwon — APPROVED** [review]
Reviewed (2).

**Pull request merged**